flipperdevices/Flipper-Android-App
# File editor for new file manager

**Background**

This PR adds file editing into experimental file manager

**Changes**

- Move part of upload module to KMP
- Extract file Uploader into decompose Composable content, which is now shared
- Add file editing - with HEX/TXT
- Save into editable file, save as new file

**Test plan**

- Open bridge connection sample
- Create or find a file
- Click on file
- Write some text in hex/txt
- Save file
- See saved file changed
